crowbar_framework/lib/crowbar/conduit_resolver.rb

```diff
@@ -106,7 +106,7 @@ def if_speed_remap
 
         sorted_ifs.each do |intf|
           speeds = cr_ohai_network[intf]["speeds"]
-          speeds = ["1g"] unless speeds # legacy object support
+          speeds = ["1g"] if speeds.nil? || speeds.empty? # legacy object support
```

Comment:
```blank?``` will not work here anyways. It's rails specific. This code can run outside rails (as it is also included in the barclamp chef recipe) via a symlink.